src/server/getters/getHackerForActiveHackathonFromSession.ts

```diff
@@ -17,21 +19,27 @@ const getHackerForActiveHackathonFromSession =
   async (): Promise<HackerFromSessionData> => {
```

Comment:
Btw this function is really hard to read but that's my fault 😄 Would you mind adding a couple comments inside to navigate the reader?